data/pathfinder/paizo/roleplaying_game/pathfinder_unchained/pu_abilities_class.lst

```diff
@@ -31,23 +31,53 @@ Wound Threshold										SORTKEY:0	CATEGORY:Special Ability	TYPE:SpecialQuality
+Legendary Mind 2		KEY:Legendary Gifts ~ Legendary Mind 2		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression		PREABILITY:1,CATEGORY=Special Ability,Legendary Gifts ~ Legendary Mind																				BONUS:VAR|Pool_ABPMentalProwess|1			DESC:Increase your bonuses from mental prowess to +6/+6/+6. You must already have legendary mind to select this legendary gift.																																																								SOURCEPAGE:p.157		ASPECT:ChildAbility|Legendary Gifts
+Legendary Shieldmaster	KEY:Legendary Gifts ~ Legendary Shieldmaster	CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression															MULT:YES		STACK:YES		CHOOSE:NOCHOICE																		DESC:Gain a +5 enhancement bonus from armor attunement for both your armor and your shield. You can select this legendary gift multiple times, choosing a different attuned suit of armor or shield each time.																																											SOURCEPAGE:p.157		ASPECT:ChildAbility|Legendary Gifts
+Legendary Twin Weapons	KEY:Legendary Gifts ~ Legendary Twin Weapons	CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression															MULT:YES		STACK:YES		CHOOSE:NOCHOICE																		DESC:Gain a +5 enhancement bonus from weapon attunement for two weapons at the same time. This counts as two legendary gifts. You can select this legendary gift multiple times, adding an additional attuned weapon with a +5 enhancement bonus each time.																																	COST:2		SOURCEPAGE:p.157		ASPECT:ChildAbility|Legendary Gifts
+Legendary Weapon		KEY:Legendary Gifts ~ Legendary Weapon		CATEGORY:Special Ability		TYPE:ABP Legendary Gifts.SpecialQuality.Automatic Bonus Progression															MULT:YES		STACK:YES		CHOOSE:NOCHOICE										BONUS:VAR|ABP_WeaponBonus|1				DESC:Your enhancement bonus from weapon attunement increases by 1. If you are attuned to more than one weapon, you can increase only one weapon's enhancement bonus in this way. The enhancement bonus on a single weapon can't exceed +5, but you can use the excess to add magic abilities to weapons. You can select this legendary gift multiple times, and it stacks up to +10 for any one weapon.												SOURCEPAGE:p.157		ASPECT:ChildAbility|Legendary Gifts
+
+###Block: Legendary Inherent Bonuses
+# Ability Name	Category of Ability		Type				Visible		Stackable?	Multiple?	Stat bonus
++1 Strength		KEY:ABP ~ +1 Strength			CATEGORY:Special Ability	TYPE:ABP Legendary Ability	VISIBLE:DISPLAY	STACK:YES	MULT:YES	CHOOSE:NOCHOICE	BONUS:STAT|STR|1|TYPE=Inherent.REPLACE
```

Comment:
Looking forward, I would rather have the stat with type using a var in a master ability, and have this bonus the variable. That would seem cleaner, and allows for easier debugging. I took great pains to refactor everything from the .REPLACE model.

Reply:
Ok.